### DISCUSSION TITLE
Invalid Validation Token

### DISCUSSION THREAD

**Discussion by gagnonraphael** [requarks/wiki#6318] [Help / Questions]
Hi !

When a user tries to register his account, he receives an email with a validation link. When he clicks on it, he always gets
`Oops, something went wrong...
Invalid validation token.`

Same problem when we use the "forget password"

I also get this error in the console `Failed to load resource: the server responded with a status of 500 ()`

**Comment by stellan-s** [comment]
In my case the problem was that outlook safe links wrapped the link and effectively destroyed the token. The devs could consider adding the link in plain text for the user to copy and paste, as a workaround.

You will probably be able to whitelist the url if they are wrapped as a safe link, however I haven't successfully tried that yet.

**Reply by iSenne** [reply]
> In my case the problem was that outlook safe links wrapped the link and effectively destroyed the token. The devs could consider adding the link in plain text for the user to copy and paste, as a workaround.

Could the issue be that 'Safe Links' is interfering with the token? I suspect that since Outlook visits the link automatically, it might be verifying the account in the background. This could explain why my accounts can log in without manually clicking the activation link.

**Comment by gagnonraphael** [comment]
You are right !
I was able to make it work by decoding the URL with https://o365atp.com/

**Reply by stellan-s** [reply]
Awesome! And thanks for the link!

**Comment by PencilAmazing** [comment]
I believe the issue is email link scanning (outlook in our case) clicks and consumes the token. Could we have a setting to allow the token to be reused, and instead have a 15 minute (or 30, emails take a while to arrive) expiry time instead?
